formulas/mo/files/mo

```diff
@@ -75,7 +75,7 @@ mo() (
                         ;;
 
                     --source=*)
-                        f2source="${1#--source=}"
+                        f2source="${arg#--source=}"
```

Comment:
I don't understand how this can change without something earlier setting arg=$1 if this is just me not getting it that's fine. Otherwise it may need fixing.